### ISSUE TITLE
Not working on Angular 9

### ISSUE THREAD

**Issue by jaytimbadia** [ngstack/code-editor#414]
ERROR in node_modules/@ngstack/code-editor/lib/code-editor.module.d.ts:5:22 - error NG6002: Appears in the NgModule.imports of AppModule, but could not be resolved to an NgModule class.

    This likely means that the library (@ngstack/code-editor) which declares CodeEditorModule has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.

    5 export declare class CodeEditorModule {
                           ~~~~~~~~~~~~~~~~


configuration:

"devDependencies": {
    "@angular-devkit/build-angular": "~0.901.1",
    "@angular/cli": "~9.1.1",
    "@angular/compiler-cli": "~9.1.1",
    "@angular/language-service": "~9.1.1",
    "@types/node": "^12.11.1",
    "@types/jasmine": "~3.5.0",
    "@types/jasminewd2": "~2.0.3",
    "codelyzer": "^5.1.2",
    "jasmine-core": "~3.5.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.4.1",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage-istanbul-reporter": "~2.1.0",
    "karma-jasmine": "~3.0.1",
    "karma-jasmine-html-reporter": "^1.4.2",
    "protractor": "~5.4.3",
    "ts-node": "~8.3.0",
    "tslint": "~6.1.0",
    "typescript": "~3.8.3"


Please let me know is it compatible with angular 9? If no, then which editor do you suggest?

**Comment by DenysVuika** [comment]
Check the Releases page. Last two releases should support Angular 10, just take previous one. There was a major version bump for Angular 10 btw, so you probably upgraded to latest

**Comment by jaytimbadia** [comment]
I download'ed the zip and ran npm install for version 2.0.0, On ng serve its still giving this error.
I now have correct version too, right, still the problem.

Please help.


-----------------------------------------------  

ERROR in node_modules/@ngstack/code-editor/lib/code-editor.module.d.ts:5:22 - error NG6002: Appears in the NgModule.imports of AppModule, but could not be resolved to an NgModule class.

    This likely means that the library (@ngstack/code-editor) which declares CodeEditorModule has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.

    5 export declare class CodeEditorModule {

**Comment by DenysVuika** [comment]
As I've said, 2.x is for Angular 10, you need 1.x or just upgrade your app to Angular 10

**Comment by jaytimbadia** [comment]
I dont understand this attitude of closing the issue.
I told you in plain english that I have installed 2.x only and my app is in angualar 10, still, its not working.

If you cant solve it, please tell that up front.
Now its closed.

**Comment by DenysVuika** [comment]
There are 2 angular 10 projects coming with the source code and both working fine. Feel free to provide a PR if you want to enhance or change something.

**Comment by cosminv6u** [comment]
It worked for me fine. 

"@angular/core": "~10.0.12",
"@ngstack/code-editor": "^2.1.0",

**Comment by LiamKenneth** [comment]
@jaytimbadia add postinstall to your scripts object in package.json
```
"scripts": {
    "postinstall": "ngcc"
}
```
Upgrade to Angular 10.

This worked for me.

**Comment by fwitkowski** [comment]
I have probably same problem.
```
"@angular/cli": "^9.1.8",
"@ngstack/code-editor": "^1.0.0",
```

in Module.ts
```
CodeEditorModule.forRoot({
      baseUrl: 'assets/monaco'
    })
```

Using code from first example:
```
<ngs-code-editor
                    [theme]="theme"
                    [codeModel]="codeModel"
                    [options]="options"
                    (valueChanged)="onCodeChanged($event)"
                >
                </ngs-code-editor>
```

```
codeModel: CodeModel = {
    language: 'json',
    uri: 'main.json',
    value: '{}',
  };

  options = {
    contextmenu: true,
    minimap: {
      enabled: true,
    },
  };

  onCodeChanged(value) {
    console.log('CODE', value);
  }
```

No errors, but I only see black box (no content, not editable)

**Comment by virux99** [comment]
i am facing this issue
```
ERROR TypeError: Cannot read property 'monaco' of null
    at SafeSubscriber._next (ngstack-code-editor.js:109)
    at SafeSubscriber.__tryOrUnsub (Subscriber.js:183)
    at SafeSubscriber.next (Subscriber.js:122)
    at Subscriber._next (Subscriber.js:72)
    at Subscriber.next (Subscriber.js:49)
    at BehaviorSubject._subscribe (BehaviorSubject.js:14)
    at BehaviorSubject._trySubscribe (Observable.js:42)
    at BehaviorSubject._trySubscribe (Subject.js:81)
    at BehaviorSubject.subscribe (Observable.js:28)
    at new TypescriptDefaultsService (ngstack-code-editor.js:108)
```

**Comment by DenysVuika** [comment]
Use 1.x version or upgrade to Angular 10 and use 2.x version. This has been stated multiple times already.
